services/agents/mongodb.go

```diff
@@ -25,11 +25,23 @@ import (
 	"github.com/percona/pmm/api/agentpb"
 	"github.com/percona/pmm/api/inventorypb"
 
+	"github.com/percona/pmm/version"
+
 	"github.com/percona/pmm-managed/models"
 )
 
 // mongodbExporterConfig returns desired configuration of mongodb_exporter process.
-func mongodbExporterConfig(service *models.Service, exporter *models.Agent, redactMode redactMode) *agentpb.SetStateRequest_AgentProcess {
+func mongodbExporterConfig(service *models.Service, exporter *models.Agent, redactMode redactMode, v *version.Parsed) *agentpb.SetStateRequest_AgentProcess {
```

Comment:
```suggestion
func mongodbExporterConfig(service *models.Service, exporter *models.Agent, redactMode redactMode, pmmAgentVersion *version.Parsed) *agentpb.SetStateRequest_AgentProcess {
```

Reply:
should we pass *version.Parsed or just a string?

Reply:
if we send a string, we must validate the parsing result inside the function and change the signature to also return the "possible" error. That's why I am passing the already parsed version